javers-core/src/main/java/org/javers/core/metamodel/object/CdoSnapshot.java

```diff
@@ -26,6 +26,7 @@
     private final CdoSnapshotState state;
     private final SnapshotType type;
     private final List<String> changed;
+    private final Long version;
```

Comment:
True, we need to write contribution guidelines.  Just push another commit but wait a while,  i will take a deeper look at this PR at the evening